src/tests/suite/SConscript

```diff
@@ -29,32 +29,47 @@ int main(int argc, char **argv)
 """
 
 def CheckCmockaSkip(context):
+    """Configure check for cmocka bug"""
     context.Message('Checking if cmocka skip() bug is present ... ')
     rc = context.TryCompile(test_cmocka_skip, '.c')
     if rc == 0:
-        print(" (Compile failed) assuming ",
-        context.Result(not rc))
+        print(" (Compile failed) assuming ", context.Result(not rc))
         return rc
     rc = context.TryLink(test_cmocka_skip, '.c')
     if rc == 0:
-        print(" (Link failed) assuming ",
-        context.Result(not rc))
+        print(" (Link failed) assuming ", context.Result(not rc))
         return rc
     prog = context.lastTarget
     pname = prog.get_abspath()
     rc = subprocess.call(pname, env={"CMOCKA_TEST_ABORT": "1"}, stdout=DEVNULL,
                          stderr=DEVNULL)
-    """ in case of abort rc is -6 instead of 134 (128+6) with shell ... """
+    #in case of abort rc is -6 instead of 134 (128+6) with shell ...
```

Comment:
Hmmm there should be some logic/convention for these '#' vs """ usage that you need to explain me ;-)

Reply:
The """ comment is used for the function documentation and is allowed there...but elsewhere, pylint complains about it.